packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -349,19 +364,23 @@ export class CodefiTokenPricesServiceV2
     currency: SupportedCurrency;
   }): Promise<Partial<TokenPricesByTokenAddress<Hex, SupportedCurrency>>> {
     const chainIdAsNumber = hexToNumber(chainId);
+    const ZERO_ADDRESS: `0x${string}` =
+      '0x0000000000000000000000000000000000000000';
 
     const url = new URL(`${BASE_URL}/chains/${chainIdAsNumber}/spot-prices`);
-    url.searchParams.append('tokenAddresses', tokenAddresses.join(','));
+    url.searchParams.append(
+      'tokenAddresses',
+      [ZERO_ADDRESS, ...tokenAddresses].join(','),
+    );
     url.searchParams.append('vsCurrency', currency);
+    url.searchParams.append('includeMarketData', 'true');
 
-    const pricesByCurrencyByTokenAddress: SpotPricesEndpointData<
-      Lowercase<Hex>,
-      Lowercase<SupportedCurrency>
-    > = await this.#tokenPricePolicy.execute(() =>
-      handleFetch(url, { headers: { 'Cache-Control': 'no-cache' } }),
-    );
+    const pricesByCurrencyByTokenAddress: AddressCryptoDataMap =
```

Comment:
Is this an accurate name for this data? Perhaps we should follow the name of the type here, so, something like this (or whatever we settle on as the name of the type):

``` suggestion
    const addressCryptoDataMap: AddressCryptoDataMap =